spec/services/candidate_interface/submit_application_choice_spec.rb

```diff
@@ -23,10 +23,11 @@
       let(:application_choice) { create(:application_choice, :unsubmitted) }
 
       it 'updates timestamps relevant to submitting an application' do
-        travel_temporarily_to(Time.zone.local(0)) do
+        submitted_at_time = 5.minutes.ago
```

Comment:
Time.zone.local(0) doesn't map to a timetable, which is why this test needed updated